gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -172,42 +168,23 @@ private static void configureErrorProneOptions(
             // Don't attempt to cache since it won't capture the source files that might be modified
             javaCompile.getOutputs().cacheIf(t -> false);
 
-            Optional<SourceSet> maybeSourceSet = project
-                    .getConvention()
-                    .getPlugin(JavaPluginConvention.class)
-                    .getSourceSets()
-                    .matching(ss -> javaCompile.getName().equals(ss.getCompileJavaTaskName()))
-                    .stream()
-                    .collect(MoreCollectors.toOptional());
-
-            // TODO(gatesn): Is there a way to discover error-prone checks?
-            // Maybe service-load from a ClassLoader configured with annotation processor path?
-            // https://github.com/google/error-prone/pull/947
             errorProneOptions.getErrorproneArgumentProviders().add(new CommandLineArgumentProvider() {
                 // intentionally not using a lambda to reduce gradle warnings
                 @Override
                 public Iterable<String> asArguments() {
-                    Optional<List<String>> specificChecks = getSpecificErrorProneChecks(project);
-                    if (specificChecks.isPresent()) {
-                        List<String> errorProneChecks = specificChecks.get();
-                        // Work around https://github.com/google/error-prone/issues/3908 by explicitly enabling any
-                        // check we want to use patch checks for (ensuring it is not disabled); if this is fixed, the
-                        // -Xep:*:ERROR arguments could be removed
-                        return Iterables.concat(
-                                errorProneChecks.stream()
-                                        .map(checkName -> "-Xep:" + checkName + ":ERROR")
-                                        .collect(Collectors.toList()),
```

Comment:
I was expecting this change to [fail this test](https://github.com/palantir/gradle-baseline/blob/f9539d53d1a1bfd945e9fc9f1db8b499e73cdbf1/gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineErrorProneIntegrationTest.groovy#L208-L238), but it is actually working 🤔

Reply:
I think the tests in `BaselineErrorProneIntegrationTest` are very broken, they pass if stop disabling the checks.

Reply:
I recall some configuration ordering issues where checks may be configured before the gradle-baseline defaults are applied and override error-prone settings